schema.sql

```diff
@@ -57,4 +57,12 @@ CREATE TABLE visits (
     date_of_visit DATE,
     FOREIGN KEY (animal_id) REFERENCES animals(id),
     FOREIGN KEY (vet_id) REFERENCES vets(id)
-);
\ No newline at end of file
+);
+
```

Comment:
- [ ] Kindly attach the screenshot of the email from microverse for work alone permit in the pull request summary so the next reviewer will be able to proceed review of your pull request. 😊

Reply:
![screencapture-mail-google-mail-u-3-2023-08-18-12_03_32](https://github.com/porag-m06/Porag-s-Vet-Clinic-DB/assets/15729467/1edb10da-159b-46e2-a9b6-28f3472e147d)